src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_TERM = 40;
```

Comment:
The constant `DAY_TERM` is not descriptive enough and could be confused with a period of time. A better name would be `DAILY_RATE` to indicate that it represents the cost per day.